packages/multi-select-combo-box/src/vaadin-multi-select-combo-box.js

```diff
@@ -298,6 +300,15 @@ class MultiSelectComboBox extends ResizeMixin(InputControlMixin(ThemableMixin(El
         },
       },
 
+      /**
+       * True when loading items from the data provider, false otherwise.
+       */
+      loading: {
```

Comment:
This property is not readonly in the regular `vaadin-combo-box`, apparently due to external filtering support:

https://github.com/vaadin/web-components/blob/e96fc97ca28ffaf94b00eb7cdd275df3011e4612/packages/combo-box/test/external-filtering.test.js#L30-L38

Reply:
Looks like this `loading` behavior is a leftover from the initial implementation: https://github.com/vaadin/vaadin-combo-box/pull/350

> In case you want to use an asynchronous, remote data source together with [propertyname]#filteredItems#
you can also set [propertyname]#loading# to true while your data is loading to display a loading spinner.